cfme/tests/v2v/test_v2v_migrations.py

```diff
@@ -4,16 +4,16 @@
 
 from cfme import test_requirements
 from cfme.cloud.provider.openstack import OpenStackProvider
-from cfme.fixtures.templates import dportgroup_template
-from cfme.fixtures.templates import dual_disk_template
-from cfme.fixtures.templates import dual_network_template
-from cfme.fixtures.templates import rhel69_template
-from cfme.fixtures.templates import rhel7_minimal
-from cfme.fixtures.templates import ubuntu16_template
-from cfme.fixtures.templates import win10_template
-from cfme.fixtures.templates import win2012_template
-from cfme.fixtures.templates import win2016_template
-from cfme.fixtures.templates import win7_template
+from cfme.fixtures.templates import DPORTGROUP_TEMPLATE
```

Comment:
this becomes needlessly verbose, how about putting these in an enum class, so they can be imported all together.

Reply:
Done